frontend/src/components/VendorInfoNav.js

```diff
@@ -44,41 +44,43 @@ const VendorInfoNav = ({
 
   return (
     <>
-    <div className='icons'>
-    <Router>
-        <div className='info-icon'>
-          <NavLink
-            activeClassName={`${showMarkets ? '' : 'active'}`}
-            to={`/arusok/${vendorId}`}
-          >
-            <div
-              className={'image ' + (status ? 'active-page' : null)}
-              onClick={() => changeStatus(true)}
+      <div className='icons'>
```

Comment:
Használjatok mindannyian Prettier-t, ami ugyanúgy van beállítva és akkor nem lesznek ilyen change-ek. Ez csak fölösleges kognitív zaj minden PR-ben, amikor újraformázza valaki a kódot.

Reply:
valószínű, hogy az enyém cseszi szét, mert valami másik extension-nel összeakadhat, és mentésenként oda-vissza formálja, akkor ezt beállítom majd